var/spack/repos/builtin/packages/readline/package.py

```diff
@@ -18,16 +18,48 @@ class Readline(AutotoolsPackage, GNUMirrorPackage):
     # URL must remain http:// so Spack can bootstrap curl
     gnu_mirror_path = "readline/readline-8.0.tar.gz"
 
-    version("8.1.2", sha256="7589a2381a8419e68654a47623ce7dfcb756815c8fee726b98f90bf668af7bc6")
+    version("8.2", sha256="3feb7171f16a84ee82ca18a36d7b9be109a52c04f492a053331d7d1095007c35")
     version("8.1", sha256="f8ceb4ee131e3232226a17f51b164afc46cd0b9e6cef344be87c65962cb82b02")
     version("8.0", sha256="e339f51971478d369f8a053a330a190781acb9864cf4c541060f12078948e461")
     version("7.0", sha256="750d437185286f40a369e1e4f4764eda932b9459b5ec9a731628393dd3d32334")
     version("6.3", sha256="56ba6071b9462f980c5a72ab0023893b65ba6debb4eeb475d7a563dc65cafd43")
 
     depends_on("ncurses")
-    # from url=https://www.linuxfromscratch.org/patches/downloads/readline/readline-6.3-upstream_fixes-1.patch
-    # this fixes a bug that could lead to seg faults in ipython
-    patch("readline-6.3-upstream_fixes-1.patch", when="@6.3")
```

Comment:
This patch is included in one of the official patches below, so our copy is no longer needed